src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -119,6 +119,9 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
     'sharesCamera',
   ]);
 
+  const {blurredVideoStream} = useKoSubscribableChildren(selfParticipant, ['blurredVideoStream']);
+  const hasBlurredBackground = !!blurredVideoStream;
```

Comment:
This avoids having duplicated states. The participant is now the single source of truth